qiskit/circuit/qpy_serialization.py

```diff
@@ -518,6 +587,16 @@ class if it's defined in Qiskit. Otherwise it falls back to the custom
 COMPLEX = namedtuple("COMPLEX", ["real", "imag"])
 COMPLEX_PACK = "!dd"
 COMPLEX_SIZE = struct.calcsize(COMPLEX_PACK)
+# Pauli Evolution Gate
+PAULI_EVOLUTION_DEF = namedtuple(
+    "PAULI_EVOLUTION_DEF", ["operator_size", "time_type", "time_size", "synth_method_size"]
+)
```

Comment:
Is this used?  I can't tell if it should have `"standalone_op"` in it too, or if this serves a different purpose.

Reply:
Yeah this is missing `standalone_op`. It's not used in the code but the convention in the file is to defined a named tuple for the pack format so there is named attribute access. But personally I just use the index normally

Reply:
```suggestion
PAULI_EVOLUTION_DEF = namedtuple(
    "PAULI_EVOLUTION_DEF", ["operator_size", "standalone_op", "time_type", "time_size", "synth_method_size"]
)
```